get-influxdb-data.sh

```diff
@@ -44,13 +44,14 @@ typeset -r INFLUXDB_QUERY_WHERE_DFLT='time > now() - 1d'
 typeset -r INFLUXDB_QUERY_GROUP_DFLT='time(1ms), "displayName"'
 typeset -r INFLUXDB_QUERY_FILL_DFLT="none"
 typeset -i DAYS_DFLT=1
+typeset -r OUTPUTFILE_DFLT="/dev/stdout"
```

Comment:
Portability: this doesn't always work, as  `/dev/stdout` is not a standard POSIX feature. Need to detect the system we're running on and set this appropriately. (Or change how we call curl.)